src/main/java/ru/mail/polis/homework/collections/mail/Salary.java

```diff
@@ -3,5 +3,9 @@
 /**
  * 1 тугрик
  */
-public class Salary {
+public class Salary extends Mail<Integer> {
+
+    public Salary(Integer message, String sender, String recipient) {
```

Comment:
то что `Mail<Integer>` - это ок, там по-другому не сделать, но в остальных местах лучше использовать примитив int, иначе может получиться ситуация, что тебе сюда вместо зарплаты передадут null и программа упадет

Reply:
чего не поправил то? должно быть: 
```suggestion
    public Salary(int message, String sender, String recipient) {
```